create-version-tags.py

```diff
@@ -0,0 +1,25 @@
+# Used by cloudbuild.yaml to tag images with correct versions based on git version tags.
```

Comment:
Please move file into a `ci/bin` or `ci` directory.

Reply:
Moved to ci/ directory